platformio.ini

```diff
@@ -11,7 +11,6 @@
 [env]
 monitor_speed = 115200 ; must remain at 115200 for improv
 board_build.filesystem = littlefs
-upload_port = /dev/ttyUSB0
```

Comment:
Squash this commit with the "New web page" commit

Reply:
I get errors every time I attempt that.  The problem (I think) is that I merged main into this branch after the "New web page" commit.  I can squash it with the apple-touch-icon commit.  But that feels somewhat ugly.

Reply:
First fix the conflicts
make sure you sync the main branch on your fork.  then on your branch do a:
`git rebase main`

then fix the 2 conflicts

then to do the squash on the platform.ini file you can do a:
`git rebase -i main`
move this commit right after the "new web page" commit and change it to squash


or you could just amend the original commit

Reply:
I did a pr on your repo showing it corrected

Reply:
If you want, we can just merge that one.  The commits retained their original authors